build/Build.cs

```diff
@@ -0,0 +1,616 @@
+﻿using System;
```

Comment:
This file is the analogue of "build.cake". Most steps (nuke "targets") should be similarly named to their equivalent build.cake step.